tech/database/cassandra/cassandra_installation.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Cassandra
+subsection: cassandra
+section: tech-database
+description: OpenSource database server for high-scale application.
+---
+
+# Apache Cassandra
+
+The Apache Cassandra database is the right choice when you need scalability and
+high availability without compromising performance. Linear scalability and proven
+fault-tolerance on commodity hardware or cloud infrastructure make it the perfect
+platform for mission-critical data.
+
+Cassandra is a partitioned row store. Rows are organized into tables with
+a required primary key. Partitioning means that Cassandra can distribute your
+data across multiple machines in an application-transparent matter. Cassandra
+will automatically re-partition as machines are added/removed from the cluster.
+Row store means that like relational databases, Cassandra organizes data by
+rows and columns. The Cassandra Query Language (CQL) is a close relative of SQL.
+
+## Quick start
```

Comment:
All of content here in developer portal should be a "Quick start" or a reference to some documentation. Not a documentation itself.